### PR TITLE
Doc 328 add known issues in prod to the docs

### PR DESCRIPTION
**Pull request by sunitaprakash**
# What does this PR do?
I have added the first draft of the Known Issues page 
Explain purpose of PR and any useful context.

### CODE REVIEW

**Comment by oojo12** [comment]
@sunitaprakash you should be using the reviewers option not assigning. If assigning it should be assigned to you since it is your PR and you are owning it. Unless @essiequoi you disagree

File: docs/platform/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+description: known issue in the Grid Platform
+---
+
+# Known Issues
+
+## Known Issues
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+Currently users on Grid Cloud and BYOC are able to create runs that resolve to the same name. This prohibits certain operations against runs/experiments.
```
Comment (oojo12) [inline docs/platform/known-issues.md:10]: maybe also make bulleted list to be consistent with artifacts

File: docs/platform/known-issues.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### [Artifacts](https://docs.grid.ai/features/runs/artifacts)
+* Artifacts don't sync for fast experiments: We've detected a race condition with short-running experiments, which may cause artifacts not to be properly synced. We're working on a long-term solution for this and will fix it in the coming days. We recommend ensuring your experiments last at least a minute (to be safe), and sleep if needed as a workaround. 
+* Canceling download of Artifacts: When downloading the artifacts, Ctrl-C does not cancel the download. You can use Ctrl-Z followed  by kill -9 $(jobs -p) as a workaround.
+* Artifacts are not saved in a subdirectory: Grid artifacts are not saved in a subdirectory for run name and experiment name. This will overwrite files if two experiments have artifacts with the same file name. Workaround would be to put the artifacts in separate folders before downloading artifacts again.
+
+### [Collaboration](https://docs.grid.ai/platform/collaboration)
+Downloading artifacts and viewing logs from a teammate's experiments is not supported from the CLI. Please use the UI as a workaround.
```
Comment (oojo12) [inline docs/platform/known-issues.md:18]: Same comment as above it applies to the rest of the document. Not an approval breaker but it is a cosmetic change we may want.

**Review by oojo12 — APPROVED** [review]
Overall content is good. But formatting could be made consistent.

**Comment by oojo12** [comment]
Approved upon condition that the -DOC-XXX is removed from the file

File: docs/platform/known-issues.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+description: known issue in the Grid Platform
+---
+
+# Known Issues
+#### Last Updated: April 1, 2022
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+* Creating runs with the same name across two cluster is currenly supported, but prohibits certain operations against runs/experiments.
```
Comment (essiequoi) [inline docs/platform/known-issues.md:9]: what specific operations are prohibited?

File: docs/platform/known-issues.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# Known Issues
+#### Last Updated: April 1, 2022
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+* Creating runs with the same name across two cluster is currenly supported, but prohibits certain operations against runs/experiments. 
+
+### [Artifacts](https://docs.grid.ai/features/runs/artifacts)
+* Artifacts don't sync for fast experiments: We've detected a race condition with short-running experiments, which may cause artifacts not to be properly synced. We recommend ensuring your experiments last at least a minute (to be safe), and sleep if needed as a workaround. 
+* Canceling download of Artifacts: When downloading the artifacts, Ctrl-C does not cancel the download. As a workaround, use Ctrl-Z followed by `kill -9 $(jobs -p)`.
+* Artifacts are not saved in a subdirectory: Grid artifacts are not saved in a subdirectory for run name and experiment name. This will overwrite files if two experiments have artifacts with the same file name. Workaround would be to put the artifacts in separate folders before downloading artifacts again.
```
Comment (essiequoi) [inline docs/platform/known-issues.md:14]: not sure what this means. can you clarify? we want to make sure this is clear from a user perspective

File: docs/platform/known-issues.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### [Collaboration](https://docs.grid.ai/platform/collaboration)
+* Downloading artifacts and/or viewing logs from a teammate's experiments is not supported from the CLI. Please use the UI as a workaround.
+
+### [Datastore](https://docs.grid.ai/features/datastores)
+* grid datastore [create](https://docs.grid.ai/features/datastores/create) ./cifar10
+Datastore upload for hard link work, but not symlink in datastore.
```
Comment (essiequoi) [inline docs/platform/known-issues.md:21]: @sunitaprakash what are we trying to tell users here?

File: docs/platform/known-issues.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### [Runs](https://docs.grid.ai/features/runs)
+* For runs with CPU/GPU specify --cpu/--gpu flag: When you run an experiment on CPU instance type, use the --cpu flag with 1 less cpu than maximum CPUs on the instance or the Run will only use 1 CPU for the experiment.
```
Comment (essiequoi) [inline docs/platform/known-issues.md:24]: this needs to be rewritten for clarity

File: docs/platform/known-issues.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+Datastore upload for hard link work, but not symlink in datastore.
+
+### [Runs](https://docs.grid.ai/features/runs)
+* For runs with CPU/GPU specify --cpu/--gpu flag: When you run an experiment on CPU instance type, use the --cpu flag with 1 less cpu than maximum CPUs on the instance or the Run will only use 1 CPU for the experiment. 
+* Folder size for [--localdir](https://docs.grid.ai/features/runs/localdir): When using the `--localdir` option the folder sizes are recommended to stay under 50MB
+
+### [Session](https://docs.grid.ai/features/sessions)
+* Cloning a Github repo with SSH is currenlty not supported. Insead, use Github's HTTPS clone method. 
+
+### Grid Ignore
```
Comment (essiequoi) [inline docs/platform/known-issues.md:30]: we need to document Grid Ignore

**Review by essiequoi — CHANGES_REQUESTED** [review]

File: docs/platform/known-issues.md (hunk 7 above)
Comment (oojo12) [reply docs/platform/known-issues.md:30]: @essiequoi i already documented .gridignore https://docs.grid.ai/features/runs in the notes.

File: docs/platform/known-issues.md (hunk 4 above)
Comment (sunitaprakash) [reply docs/platform/known-issues.md:14]: There are 3 different know issues with Artifacts 
1. Artifacts don't sync for fast experiments
2. Canceling download of Artifacts
3. Artifacts are not saved in a subdirectory

File: docs/platform/known-issues.md (hunk 3 above)
Comment (sunitaprakash) [reply docs/platform/known-issues.md:9]: @oojo12 Can you add more information for this ??

File: docs/platform/known-issues.md (hunk 5 above)
Comment (sunitaprakash) [reply docs/platform/known-issues.md]: Updated this with more details.

**Review by essiequoi — APPROVED** [review]
Approving for now, to be iterated on

**Pull request merged**